AY2021S2-CS2103-T14-1/tp
# Update FindRoomCommand to also search for matching tags

### What this does
Updates the logic of FindRoomCommand to also search by tags. A separate command was not done as room numbers will differ greatly from tags, likely not causing to overlap between room number and tags. Within the context of SunRez, this implementation should be sufficient when filtering rooms with a keyword.
Closes #93

### How to test
1. Create a room with a tag
2. Run `ofind [tag name or partial tag name]`
3. Verify that the room you added with the tag shows up
4. Run `ofind [keyword not in tag or room number]`
5. Verify that results are empty

### What is left
1. Tests
2. Update UG and DG
